.github/workflows/main.yml

```diff
@@ -310,3 +310,8 @@ jobs:
     - name: Test R
       run: |
         python tests/ci_build/test_r_package.py --compiler="${{ matrix.config.compiler }}" --build-tool="${{ matrix.config.build }}"
+
+    - name: Check R
+      run: |
+        make Rbuild
+        R CMD check --as-cran xgboost*.tar.gz --no-manual
```

Comment:
@ahmedkrmn Can you move this to a new job? The `test-with-R` job is running on Windows, and it's harder to get the R check work on Windows. Make sure to set `os` to Ubuntu Linux.